web/pdf_find_bar.js

```diff
@@ -201,7 +206,32 @@ var PDFFindBar = (function PDFFindBarClosure() {
       } else {
         this.open();
       }
-    }
+    },
+
+    /**
+     * @private
+     */
+    _adjustWidth: function PDFFindBar_adjustWidth() {
+      if (!this.opened) {
+        return;
+      }
+
+      // The find bar has an absolute position and thus the browser extends
+      // its width to the maximum possible width once the find bar does not fit
+      // entirely within the window anymore (and needs to be wrapped). Here we
```

Comment:
Nit: Maybe `... (and parts/all of it is automatically wrapped)...` or something instead, since the wrapping is automatic based on the CSS.